packages/back-end/src/validators/experiment-warnings.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+
+export const autoUpdateFailed = z
+  .object({
+    type: z.literal("auto-update"),
+    success: z.boolean(),
```

Comment:
I should have caught this earlier, but we should not be firing a `warning` event when a snapshot update is successful.  Successful snapshots can happen hundreds of times a day for some orgs, so coupling that to the same event as actual warnings makes it impossible to use it for a Slack notification.  There will be so much noise in the Slack channel, you won't be able to see actual warnings